.github/labels.yml

```diff
@@ -0,0 +1,54 @@
+- name: "after coreth x subnet-evm"
+  color: "98E19D"
+  description: "To be done after merging the coreth and subnet-evm codebases"
+- name: "after libevm"
+  color: "98E19D"
+  description: "To be done only after using libevm for good"
```

Comment:
[no action required] these labels share the same color

Reply:
Indeed but:

1. already the case, we may change in a subsequent PR modifying this file
2. grouping similar labels (after xyz) with the same color is useful in my opinion 😉 Especially finding a distinguishable color for each label is rather difficult as soon as you have around 10 labels 😓